wled00/bus_manager.cpp

```diff
@@ -384,24 +397,17 @@ BusPwm::BusPwm(BusConfig &bc)
   if (!IS_PWM(bc.type)) return;
   unsigned numPins = NUM_PWM_PINS(bc.type);
   _frequency = bc.frequency ? bc.frequency : WLED_PWM_FREQ;
+  // duty cycle resolution (_depth) can be extracted from this formula: CLOCK_FREQUENCY > _frequency * 2^_depth
+  _depth = uint8_t(log((float)CLOCK_FREQUENCY / (float)_frequency) / log(2.0));
```

Comment:
Why floating point and `log()`? You can get away using integer math and shifting. 
I.e. `_depth = (CLOCK_FREQUENCY/_frequency) >> 8;`
And then devise actual bit depth by counting active bits.

Or use `for` loop to get required bits by shifting dynamically.
I.e. `for (_depth=12; _depth>8; _depth--) if (((CLOCK_FREQUENCY/_frequency)>>_depth) > 0) break;`